src/eval-frame/index.jsx

```diff
@@ -25,6 +25,7 @@ import { iodide } from './iodide-api/api'
 import './port-to-editor'
 
 window.iodide = iodide
+window.FETCH_RESOLVERS = {}
```

Comment:
Do we actually need to add a property to the window? Could we not define this object in a file and then import it elsewhere?

Reply:
Yep, it would be easy to have something that actually just closes over a variable in some way, along with some functions that fetches the `resolve` and `reject` as well as deletes.

Reply:
fwiw we use this pattern elsewhere, but agreed that maybe we should move to something that doesn't attach to `window`